mtls/cli.py

```diff
@@ -202,6 +202,17 @@ def add_server(ctx, name):
         config.write(config_file)
 
 
+@server.command("list", help="List servers")
+@click.pass_context
+def list_servers(ctx):
+    config_path = ctx.obj["config_path"]
+    config = ConfigParser()
+    config.read(config_path)
+    for server_name, section in config.items():
+        if server_name != 'DEFAULT':
+            print(server_name)
```

Comment:
You should use `click.echo` here for consistency as well as it's the recommended way to output when using click https://click.palletsprojects.com/en/7.x/quickstart/?highlight=echoing#echoing